doc/lua_api.txt

```diff
@@ -7596,6 +7596,13 @@ Used by `minetest.register_node`.
         -- intensity: 1.0 = mid range of regular TNT.
         -- If defined, called when an explosion touches the node, instead of
         -- removing the node.
+        
+        mod_origin,
+        -- stores which mod registered a node
```

Comment:
Perhaps something like: 
```
Denotes the mod that was executed during the registration.
Equals to "??" when the item registration occurred outside the initial loading process.
````
Technically this would also make the last two sentences superfluous, but I'm not sure how understandable this sentence is to modders.